contracts/src/interfaces/ITeleporterTokenBridge.sol

```diff
@@ -18,16 +18,16 @@ import {ITeleporterReceiver} from "@teleporter/ITeleporterReceiver.sol";
  * @param destinationBridgeAddress address of the destination token bridge instance
  * @param recipient address of the recipient on the destination chain
  * @param primaryFee amount of tokens to pay for Teleporter fee on the source chain
- * @param secondaryFee amount of tokens to pay for Teleporter fee if a multihop is needed.
- * @param allowedRelayerAddresses addresses of relayers allowed to send the message
+ * @param secondaryFee amount of tokens to pay for Teleporter fee if a multihop is needed
+ * @param requiredGasLimit gas limit requirement for sending to a destination token bridge
```

Comment:
Can you explain that this is required because a source/destination bridge contracts are able to call arbitrary code?

Reply:
updated comment